march_rqt_note_taker/src/march_rqt_note_taker/notes_widget.py

```diff
@@ -55,3 +65,25 @@ def _delete_selected(self):
             indices = [index for index in selection_model.selectedIndexes() if not index.column()]
             if indices and indices[0].isValid():
                 self._model.remove_rows(indices[0].row(), len(indices))
+
+    def _handle_load(self):
+        rospy.logwarn('Loading notes from a file is not yet implemented')
+
+    def _handle_save(self):
+        result = QFileDialog.getSaveFileName(self, 'Save File')
+        file_name = result[0]
+        if file_name:
+            try:
+                handle = open(file_name, 'w')
+            except IOError as e:
+                rospy.logwarn('Failed to open file: {0}'.format(e))
+                return
+
+            try:
+                handle.write(str(self._model))
+            except Exception as e:
+                rospy.logwarn('Failed to write to file: {0}'.format(e))
```

Comment:
Are there errors that occur only when trying to write, and not when opening? I think I would move this into the try/except above and use with open(file_name, 'w') ...